docs/InfoListItem.md

```diff
@@ -101,3 +101,39 @@ import { Link } from 'react-router-dom';
     button
 />
 ```
+
+### `sx` Class Overrides
```

Comment:
The formatting for this table needs fixed
![Screen Shot 2022-05-16 at 9 13 43 AM](https://user-images.githubusercontent.com/13989985/168600635-295122c1-25be-402b-87f0-7f4b679aa73b.png)